webapp/tests/karma/ts/services/db.service.spec.ts

```diff
@@ -309,44 +310,40 @@ describe('Db Service', () => {
 
     for (const method in methods) {
       if (methods[method]) {
-        it(`should stub ${method}`, fakeAsync(() => {
-          const stubbedMethod = sinon.stub(window.PouchDB.prototype, method);
+        it(`should wrap ${method}`, fakeAsync(async () => {
           getService();
           const db = service.get();
 
-          expect(stubbedMethod.callCount).to.equal(0);
-
-          methods[method].forEach(({ args }) => {
+          for (const args of methods[method]) {
             sinon.resetHistory();
-            db[method](...args);
-            expect(stubbedMethod.callCount).to.equal(1);
-            expect(stubbedMethod.args[0]).to.deep.equal(args);
-            expect(runOutsideAngular.callCount).to.equal(1);
-          });
+            try {
+              await db[method](...args.args);
+            } catch {
+              // do nothing
+            }
+            expect(runOutsideAngular.called).to.be.true;
```

Comment:
Because we don't stub anymore, the number of calls depends on the method, and can vary greatly.